review-tools/pick-to-branch

```diff
@@ -11,7 +11,14 @@ function usage {
     A release branch may be given simply as 102, 110, 111, 30, 31."
 }
 
+num=0
+
 case $# in
+3)
+    id=$1
+    b=$2
+    num=$3
+    ;;
 2)
     id=$1
     b=$2
```

Comment:
In case $# == 2 it is ambiguous from the help output whether `[<id>]` or `[<num>]` is to be provided; please clarify.

Reply:
Done